### PR TITLE
Remove unused Tags and FSRoot from NodeUp

### PR DESCRIPTION
**Pull request by hakman**
/kind cleanup

### CODE REVIEW

File: nodeup/pkg/bootstrap/install.go [hunk 1]
```diff
@@ -33,21 +32,17 @@ import (
...
 type Installation struct {
-	FSRoot          string
 	CacheDir        string
 	RunTasksOptions fi.RunTasksOptions
 	Command         []string
 }
 
 func (i *Installation) Run() error {
-	distribution, err := distros.FindDistribution(i.FSRoot)
+	_, err := distros.FindDistribution("/")
```
Comment (rifelpet) [inline nodeup/pkg/bootstrap/install.go:41]: I think we could get rid of the string argument in `FindDistribution()` since it is only ever passed `"/"`

Comment (hakman) [reply nodeup/pkg/bootstrap/install.go:41]: Maybe we want to add some tests for it in the future, or use in some container?

File: cmd/nodeup/main.go [hunk 2]
```diff
@@ -101,7 +101,6 @@ func main() {
 			i := bootstrap.Installation{
 				CacheDir: flagCacheDir,
 				Command:  command,
-				FSRoot:   flagRootFS,
```
Comment (johngmyers) [inline cmd/nodeup/main.go:104]: You should probably also remove the `rootfs` flag.

Comment (hakman) [reply cmd/nodeup/main.go:104]: Good point. Forgot about it.

**Pull request merged**